components/prism-java.js

```diff
@@ -35,7 +35,7 @@
 			lookbehind: true
 		},
 		'namespace': {
-			pattern: /\b(package\s+|import\s+(?:static\s+)?)[a-z]\w*(\.[a-z]\w*)+/,
+			pattern: /\b(package\s+|import\s+(?:static\s+)?|module\s+|requires\s+|exports\s+|uses\s+|provides\s+|open\s+|opens\s+|to\s+|with\s+|transitive\s+)[a-z]\w*(\.[a-z]\w*)+/,
```

Comment:
Please use the following instead:

```js
/(\b(?:exports|import(?:\s+static)?|module|open|opens|package|provides|requires|to|transitive|uses|with)\s+)[a-z]\w*(\.[a-z]\w*)+/
```

It's a little shorter and has the keywords in the right order.

Reply:
Thanks a lot,  the pattern section contains a shorter version now.